dist/selleckt-legacy.js

```diff
@@ -105,7 +105,7 @@ function MultiSelleckt(options){
 
 MultiSelleckt.prototype = Object.create(SingleSelleckt.prototype);
 
-MultiSelleckt.prototype._mutationHandler = function (mutations){
+MultiSelleckt.prototype._mutationHandler = function(mutations){
```

Comment:
Can you remove the changes for `dist/selleckt-legacy.js` and `dist/selleckt.js` from this PR please. The version bump should generate those files. This also reduce the risk of reviewing the wrong file, like it did 😉